src/compiler/checker.ts

```diff
@@ -2682,10 +2691,14 @@ namespace ts {
                     else {
                         symbolFromVariable = getPropertyOfVariable(targetSymbol, name.escapedText);
                     }
-
                     // if symbolFromVariable is export - get its final target
                     symbolFromVariable = resolveSymbol(symbolFromVariable, dontResolveAlias);
-                    let symbolFromModule = getExportOfModule(targetSymbol, specifier, dontResolveAlias);
+
+                    const commonJSPropertyAccess = getCommonJSPropertyAccess(node);

```

Comment:
Shouldn't all this work be under `getTargetOfAliasDeclaration` (which `resolveSymbol` eventually calls out to), and not in `getExternalModuleMember` directly?

Reply:
in the test case below, resolveSymbol is called first on the BindingElement `a`, which goes to getTargetOfImportSpecifier. Do you mean that function? I put the code in getExternalModuleMember because it seemed similar to the `export=` code above, and because `getExternalModuleMember` is called several places.

But looking again, it's only two other places, both exports where this wouldn't apply.